code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/GithubOAuthImpl.java

```diff
@@ -41,7 +42,7 @@ public Account getAccount(ApiRequest request) {
             return null;
         }
         String token = request.getServletContext().getRequest().getHeader(AUTH_HEADER);
-        if (StringUtils.isEmpty(token)) {
+        if (StringUtils.isEmpty(token) || !token.startsWith(AUTH_TYPE)) {
```

Comment:
If there are basic creds and no token query param it will end up in basicauthimpl.

The reason it happens is if you use API keys in the API UI the browser remembers the basic auth creds and sends it with other requests.  The UI usually overrides this with a bearer jwt, but /v1/subscribe is a websocket connection where I can't send arbitrary http headers.  So we put the token in the query instead, and basic gets sent too, and the net result is that no resource.change events get sent to update the UI as things change.